src/main/webapp/3Dscript.js

```diff
@@ -18,6 +18,13 @@ const poseTransform = {
   /* Scalar*/ scale: 1};
 const apiKey = 'AIzaSyDCgKca9sLuoQ9xQDfHUvZf1_KAv06SoTU';
 
+/* Matrices used to scale and unscale pose cylinders */
```

Comment:
change the scale factor to a variable constant so the inverse and original matrix always keep this property.